src/ucp/api/ucp.h

```diff
@@ -853,7 +853,11 @@ ucs_status_t ucp_worker_wait(ucp_worker_h worker);
  *
  * @param [in]  worker    Worker of notified events.
  *
- * @return Error code as defined by @ref ucs_status_t
+ * @return ::UCS_OK        The operation completed successfully. File descriptor
+ *                         will be signaled by new events.
+ * @return ::UCS_ERR_BUSY  Some events arrived before the operation and has
```

Comment:
@yosefe The `UCS_ERR_BUSY` status doesn't mean that `fd` is not armed. The `fd` is armed successfully and ready for other events arrival signal. The `fd` will be triggered If some event arrived just after arm() returns `UCS_ERR_BUSY`.
This status just mean that we had some `There are unprocessed events` which should be processed `by calling ucp_worker_progress()` and these events will not trigger `fd` on `epoll`.